moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -32,7 +32,7 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Author: Ioan Sucan */
+/* Author: Ioan Sucan, Jens Petit */
```

Comment:
What did you change here?

Reply:
I think I added it because of the documentation PR #1645 which was originally included in the feature branch but than dropped after a rebase. However, these are comments only, I will remove me again from the author list. I will create a separate PR as I can't push directly to the feature branch.